privaterelay/templates/includes/dashboard-filter.html

```diff
@@ -0,0 +1,66 @@
+{% load static %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+<div class="c-filter">
+    <div class="c-filter-search">
+        <div class="c-filter-icon">
+            <img src="/static/images/icon-search.svg" alt="">
+        </div>
+        <form action="">
+            <input class="c-filter-search-input js-filter-search-input " type="search" name="search" id="search">
+            <div class="c-filter-case-count">
+                <span class="js-filter-case-visible"></span>
+                /
+                <span class="js-filter-case-total"></span>
+            </div>
+            <button class="c-filter-reset js-filter-reset">
+                <img src="/static/images/x-close.svg" alt="">
+            </button>
+        </form>
+    </div>
+    <div class="c-filter-date">
+        <div class="c-filter-icon">
+            <img src="/static/images/icon-calendar.svg" alt="">
+        </div>
+    </div>
+    <div class="c-filter-cateogry">
+        <div class="c-filter-icon">
+            <img src="/static/images/icon-filter.svg" alt="">
+        </div>
+    </div>
+    <div class="c-filter-action">
+        <!-- Button -->
+        <form action="/emails/" class="dash-create" method="POST">
+            <input type="hidden" name="api_token" value="{{ user_profile.api_token }}">
+            {% if user_profile.at_max_free_aliases and not user_profile.has_unlimited %}
+            <a
+                href="{{ settings.FXA_SUBSCRIPTIONS_URL }}/products/{{ settings.PREMIUM_PROD_ID }}?plan={{ settings.PREMIUM_PRICE_ID }}"
+                target="_blank"
+                rel="noopener noreferrer""
+                class="btn btn-blue--ghost"
+                data-event-label="Upgrade"
+            >
+              <span class="generate-new-alias-text">
+                {% ftlmsg 'profile-label-upgrade' %}
+              </span>
+            </a>
+
+            {% else %}
+            <button
```

Comment:
Because this is in `dashboard-filter.html` and `profile.html` has this:

```
        {% if relay_addresses|length == 0 %}
          {% include "includes/dashboard_onboarding.html" %}
        {% else %}
          {% include "includes/dashboard-filter.html" %}
        {% endif %}
```

It means that there's no "Generate new alias" button on the dashboard when someone has 0 aliases. 😢 

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/71928/124659236-580bcc80-de6a-11eb-8117-dde9c7806bae.png">

Reply:
There's an entire new on-boarding flow I need to layout. We need to fix this pronto.

Reply:
There's an entirely new on-boarding comp, but this fix works for now:

![image](https://user-images.githubusercontent.com/2692333/124973192-d7c3a380-dff0-11eb-85f4-109e43e8fdeb.png)